pages/search/search.scss

```diff
@@ -1,4 +1,8 @@
 .search-page {
+  .btn-dismiss {
```

Comment:
There are so many custom/new button styles on Pulse... I don't know what the best way is to keep thing tidy anymore 😂